keps/sig-node/3085-pod-conditions-for-starting-completition-of-sandbox-creation/README.md

```diff
@@ -988,6 +1007,15 @@ https://storage.googleapis.com/k8s-triage/index.html
 We expect no non-infra related flakes in the last month as a GA graduation criteria.
 -->
 
+Tests List
+
+- [x] Pod Conditions Test (https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/pod_conditions_test.go)
```

Comment:
can you please post the link with the regex filter? 

<img width="516" alt="image" src="https://github.com/kubernetes/enhancements/assets/9950081/52e869de-0152-4e34-be50-60b8fe387fcc">

Reply:
Today I learned!  Thanks for the hint.  I updated this.  

I don't like that these tests aren't actually labelled with [NodeFeature:PodReadyToStartContainers] but the regex will catch them.